setup.py

```diff
@@ -64,7 +64,7 @@
     # Used in quizzes module for adding new quizzes.
     "django-nested-admin>=4.0,<4.1",
     # SIO2 dependencies:
-    "filetracker>=2.1,<3.0",
+    "filetracker[server]>=2.2.0,<3.0",
```

Comment:
Is it really required to add `filetracker[server]` to setup.py requirements? Production code on Szkopul requires only client code, as filetracker server is running on another system. And installing with server dependencies in prod crashed because of some version conflicts. Couldn't it be handled like in Olaf's PR https://github.com/sio2project/oioioi/pull/433 ?

Reply:
Good point! I'll remove the `[server]` optional dependencies from `setup.py` and add them to the `Dockerfile`.